src/sections/Calendar/index.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import CalendarStyleWrapper from "./calendar.style";
+import FullCalendar from "@fullcalendar/react";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import interactionPlugin from "@fullcalendar/interaction";
+import googleCalendarPlugin from "@fullcalendar/google-calendar";
+
+const Calendar = () => {
+
+    return (
+        <CalendarStyleWrapper>
+            <div>
+                <h1>Community Calendar</h1>
+                <p>Join any or all of the weekly meetings</p>
+                <div className="calendar-wrapper">
+                    <div className="calendar-grid">
+                        <FullCalendar
```

Comment:
How can the button from this FullCalendar component be overridden with the Layer5NG reusecore/button?

Reply:
@vineethvanga18, will you look into https://fullcalendar.io/docs/css-customization and see that the Layer5ng button style is applied (ideally, not copying / repeating the core button's style)?

Reply:
@leecalcote is this addressed in your commits?

Reply:
It's not. Ideally the button in `resusecore/Button` should be used directly instead of just copying the css